mozci/sources/tc.py

```
@@ -289,3 +292,19 @@ def generate_task_graph(scopes, tasks, metadata):
         'metadata': metadata
     }
     return task_graph
+
+
+def validate_graph(graph):
+    """ This function tests the graph with a JSON schema
+    """
+    schema = requests.get('http://schemas.taskcluster.net/scheduler/v1/task-graph.json').json()
+
+    # validate() does not return a value if valid, thus, not keeping track of it.
+    valid = True
+    try:
+        validate(instance=graph, schema=schema, format_checker=FormatChecker())
+    except Exception, e:
```

My apologies if I wasn't clear. Remove the try/block to let the exception raise all the way.